libs/partners/together/langchain_together/completions.py

```diff
@@ -109,7 +108,7 @@ def _format_output(self, output: dict) -> str:
 
     @staticmethod
     def get_user_agent() -> str:
-        return f"langchain-together/{__version__}"
+        return "langchain-together/0.3"
```

Comment:
let's rely on `__version__` if that works for you. Otherwise more difficult to maintain

Reply:
Makes sense, done!

Reply:
Should we add this in all methods? It's currently only there for completions

Reply:
Up to you all! If you're not using this data on your side, feel free to remove this.

Reply:
We're not using it so I'll just remove, thanks!